CONTRIBUTORS.md

```diff
@@ -299,6 +299,7 @@ Thanks for contributing!  If you want to contribute but aren't sure how, please
 1. [Yash Bhamare](https://github.com/ydb242)
 1. [Yatin Mahajan](https://github.com/yatin-mahajan)
 1. [Yeshpal Jain](https://github.com/yeshpal-jain-seagate) 
+1. [Sayed Alfhad Shah](https://github.com/alfhad)
```

Comment:
Can you please put your name in alphabetical order? Sorry I didn't mention that initially!

Reply:
Done